workspaces/orchestrator/plugins/orchestrator-backend/src/service/router.ts

```diff
@@ -724,28 +837,37 @@ function setupInternalRoutes(
         message: `Received request to '${endpoint}' endpoint`,
       });
 
-      const decision = await authorize(
-        _req,
-        orchestratorWorkflowInstanceReadPermission,
-        permissions,
-        httpAuth,
-      );
-      if (decision.result === AuthorizeResult.DENY) {
-        manageDenyAuthorization(endpointName, endpoint, _req);
-      }
-
       const includeAssessment = routerApi.v2.extractQueryParam(
         c.request,
         QUERY_PARAM_INCLUDE_ASSESSMENT,
       );
 
-      return routerApi.v2
-        .getInstanceById(instanceId, !!includeAssessment)
-        .then(result => res.status(200).json(result))
-        .catch(error => {
-          auditLogRequestError(error, endpointName, endpoint, _req);
-          next(error);
-        });
+      try {
+        const assessedInstance = await routerApi.v2.getInstanceById(
```

Comment:
I suggest to rename assessedInstance to instance
I got confused here and thought you are checking the parent assessment instance

Reply:
I can but the `assessedInstance` name is derived from its type `AssessedProcessInstanceDTO`